### DISCUSSION TITLE
Nextversion.md Issue

### DISCUSSION THREAD

**Discussion by hanan-commits-work** [iTwin/itwinjs-core#3064] [Q&A] [answered]
I noticed that the documentation for the nextversion.md is replaced by just Material Creation. Is there any alternate link for the documentation, if so then i cant find it.

**Comment by pmconne** [comment, marked as answer]
You are looking at master, which is now focused on 3.1.0. Use the release/3.0.x branch until official release of 3.0.0.

**Reply by hanan-commits-work** [reply]
Got it thank you.
